src/resources/boleto/service.js

```diff
@@ -64,7 +70,13 @@ module.exports = function boletoService ({ operationId }) {
           metadata: { boleto },
         }))
 
-    const updateBoletoStatus = ({ issuer_response_code, status, boleto_url }) => { // eslint-disable-line
+    const updateBoletoProviderResponse = ({
+      issuer_response_code, // eslint-disable-line
```

Comment:
Sugestão: é sempre bom restringir as regras de linter que estamos desabilitando 😬 

```suggestion
      issuer_response_code, // eslint-disable-line camelcase
```

Reply:
Feitoooo :heart: :rocket: